proto/zitadel/org/v2/org_service.proto

```diff
@@ -138,6 +139,45 @@ service OrganizationService {
       };
     };
   }
+
+  // Search Organizations
+  //
+  // Search for Organizations. By default, we will return all organization of the instance. Make sure to include a limit and sorting for pagination..
+  rpc ListOrganizations(ListOrganizationsRequest) returns (ListOrganizationsResponse) {
+    option (google.api.http) = {
+      post: "/v2/organizations/_search";
+      body: "*";
+    };
+
+    option (zitadel.protoc_gen_zitadel.v2.options) = {
+      auth_option: {
+        permission: "org.read"
+      }
```

Comment:
how to handle specific permissions on multiple orgs

Reply:
Changed to 'authenticated' and also added additional integration tests